Python/Product/PythonTools/PythonTools/Commands/SendToReplCommand.cs

```diff
@@ -40,24 +41,28 @@ class SendToReplCommand : Command {
             _serviceProvider = serviceProvider;
         }
 
-        public override void DoCommand(object sender, EventArgs args) {
+        public override async void DoCommand(object sender, EventArgs args) {
             var activeView = CommonPackage.GetActiveTextView(_serviceProvider);
             var project = activeView.GetProjectAtCaret(_serviceProvider);
             var analyzer = activeView.GetAnalyzerAtCaret(_serviceProvider);
 
             var repl = ExecuteInReplCommand.EnsureReplWindow(_serviceProvider, analyzer, project);
             repl.Show(true);
 
-            PythonReplEvaluator eval = repl.InteractiveWindow.Evaluator as PythonReplEvaluator;
-
-            eval.EnsureConnected();
-            repl.InteractiveWindow.Submit(GetActiveInputs(activeView, eval));
+            var version = analyzer.InterpreterFactory.Configuration.Version.ToLanguageVersion();
 
             repl.Show(true);
+
+            foreach (var code in GetActiveInputs(activeView, version)) {
```

Comment:
Why do we submit each input individually instead of sending them all to SubmitAsync at once?